### PR TITLE
Changed UI for Upload Fragment

### PR DESCRIPTION
**Pull request by dynamitechetan**
Changes: 
1. Updated Upload Screen UI to match the Material Theme.
2. Using Flat Icons

Screenshots for the change: 

<table><tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by Abhi2424shek** [comment]
Replace the Bluetooth button with a share button.

**Comment by dynamitechetan** [comment]
But currently Bluetooth sharing is functional.

**Comment by Abhi2424shek** [comment]
Theres a bit of padding issue on the left side. (Space between the pic and the card margin on right is greater than that on left)

**Comment by dynamitechetan** [comment]
@Abhi2424shek padding is equal but we have scrollbars in grid view on right side.   That's why it's looking like additional padding.

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 1]
```diff
@@ -163,6 +185,8 @@
 
     String[] service;
 
+    int color = Color.parseColor("#757575");
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: Extract this out and put it in `colors.xml`.

File: app/src/main/res/layout/upload.xml [hunk 2]
```diff
@@ -248,12 +268,29 @@
...
          android:layout_width="match_parent"
          android:layout_height="wrap_content"
-         android:numColumns="3" >
+         android:background="#fff"
+         android:numColumns="3"
+         android:layout_margin="5dp"
+         android:gravity="center">
      </GridView>
-
+</android.support.v7.widget.CardView>
```
Comment (pa1pal) [inline app/src/main/res/layout/upload.xml]: Indentation issue.

File: app/src/main/res/layout/upload.xml [hunk 3]
```diff
@@ -9,101 +9,121 @@
...
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_alignParentLeft="true"
             android:layout_centerVertical="true"
-            android:layout_marginLeft="10dp"
             android:text="@string/accounts"
-            android:textAppearance="?android:attr/textAppearanceMedium"
-            android:textColor="@color/blue_light"
-            android:textStyle="bold" />
+            android:textSize="16sp"
```
Comment (pa1pal) [inline app/src/main/res/layout/upload.xml]: Please try to add all dimensions in the `dimens.xml` as much as possible. I know this has not been the case in the current codebase, but try to follow the conventions wherever possible.

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 4]
```diff
@@ -503,10 +531,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 convertView = mInflater.inflate(R.layout.gridviewitem, null);
                 holder.imageview = (ImageView) convertView
                         .findViewById(R.id.ImageGrid);
-                holder.imageview.setMaxWidth(100);
-                holder.imageview.setMaxHeight(100);
+//                holder.imageview.setMaxWidth(100);
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: Remove unused code.

**Comment by dynamitechetan** [comment]
@pa1pal please check now

**Pull request merged**